internals/plan/plan.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021 Canonical Ltd
+// Copyright (c) 2024 Canonical Ltd
```

Comment:
This should become a range right?

Reply:
I've seen people in snapd simply updating to latest, including in Pebble, so I think this is OK. Resolving.